source/default_mode/Host.h

```diff
@@ -533,7 +533,7 @@ class Host: public Organism {
    * Purpose: To mutate a host's interaction value. This is called on newly generated
    * hosts to allow for evolution to occur.
    */
-  void mutate(){
+  void mutate(std::string mode = "vertical"){

```

Comment:
This should default to "none" for hosts since they don't have vertical transmission